packages/core/src/page.js

```diff
@@ -96,6 +101,8 @@ export default class Page extends EventEmitter {
   async close() {
     if (!this.browser) return;
 
+    this.log.debug('Page closing', this.meta);
```

Comment:
I also moved this from the close handler since the handler will be called on every automatically created page while this method is only called when intentionally closing a page. Having it remain in the former location would show several close logs in a row for pages with many frames.